composer.json

```
@@ -44,25 +43,21 @@
     "bin": [
         "bin/monorepo-tools"
     ],
+    "config": {
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
+    },
     "scripts": {
         "all": [
+            "@rector",
+            "@ecs",
             "@unit-tests",
-            "@cs-fixer",
             "@phpstan"
         ],
-        "cs-fixer": [
-            "vendor/bin/ecs check src tests --config ecs.php --fix --ansi"
-        ],
-        "phpstan": [
-            "vendor/bin/phpstan analyze src tests --level=5 --memory-limit=1G --ansi"
-        ],
-        "unit-tests": [
-            "vendor/bin/phpunit --colors=always"
-        ]
-    },
-    "config": {
-        "allow-plugins": {
-            "dealerdirect/phpcodesniffer-composer-installer": true
-        }
+        "ecs": "@php vendor/bin/ecs check --fix --ansi",
```

I’d prefer using the same name as we have in contao/contao WDYT?

```suggestion
        "cs-fixer": "@php vendor/bin/ecs check --fix --ansi",
```

---

We will soon have `ecs` there, too, because we use the tool name as key everywhere else.